ssl/s3_lib.c

```diff
@@ -4612,6 +4621,41 @@ EVP_PKEY *ssl_generate_pkey_curve(int id)
     EVP_PKEY_CTX_free(pctx);
     return pkey;
 }
+
+/* Generate a param from a group ID */
```

Comment:
I think the multi-line format for comments before a function makes it pop out more.  not a blocking comment :)